backend/mora/settings.py

```diff
@@ -44,4 +44,13 @@
 
 PROD_MODE = False
 
+try:
+    with open('backend/mora/user_settings.json') as f:
```

Comment:
The path for `user_settings` should optionally be configurable, e.g. with an environment variable.
This is in part a wish from the ops people, so the config file can exist separately from the code, which makes deployment and maintenance slightly easier.

(Ideally, we should do this for the location of the main settings file as well, but I think that's out of scope for this branch)